src/viking.js

```diff
@@ -1,11 +1,64 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health
+        this.strength = strength
+    }
+    attack() {
+        return this.strength
+    }
+    receiveDamage(damage){
+       this.health = (this.health - damage)  // "should remove the received damage from the health property"
+    }
+};
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor (name, health, strength) {
+    super (health, strength) // the ones that are in the "mother" class 
+    this.name = name 
+    }
+    receiveDamage(damage){
+       this.health = (this.health - damage)    // we need to repeat this one (and not the attack()etc because the conditions are part of this step
+       if (this.health > 0) return `${this.name} has received ${damage} points of damage`
+       if (this.health === 0) return `${this.name} has died in act of combat`
```

Comment:
Instead of another `if` statement, you should use the `else`, because the health can be less than 0, if the attack is much bigger than the health.
```
if ( this.health > 0 ) return `${this.name} has received ${damage} points of damage`
 else return `${this.name} has died in act of combat`;
```
It's better to use the `else`, because it will take care of every event that doesn't match that `if` condition above.